core/templates/dev/head/domain/learner_dashboard/learner_dashboard_icons_directive.html

```diff
@@ -21,5 +21,13 @@
 <i class="oppia-learner-dashboard-icon fa fa-spinner"
    ng-if="belongsToIncompleteActivities()"
    aria-hidden="true"
+   tooltip-append-to-body="true"
+   tooltip-class="library-incomplete-activity-tooltip"
    uib-tooltip="<['I18N_LIBRARY_INCOMPLETE_ACTIVITY_ICON' | translate]>"
    tooltip-placement="left"></i>
+<style>
+  .library-incomplete-activity-tooltip .tooltip-inner {
+    max-width: none;
+    white-space: nowrap;
```

Comment:
@srikar0896 Although its not needed, the changes you've made allow the "Text not limited to single line will look like this" (as mentioned in the screenshots provided above).

Reply:
@ishucr7 the above style tag is for showing the text inside the tooltip in a single line.I have posted the screenshot just to show how it looks when the text is not limited to single line.Thanks.